components/rich_text_editor/extensions/emoji/EmojiList.vue

```diff
@@ -0,0 +1,118 @@
+<template>
+  <div class="d-p4 d-bar8 d-bgc-contrast d-fs-200 d-truncate">
+    <template v-if="items.length">
+      <button
+        v-for="(item, index) in items"
+        :key="index"
+        :class="[
+          'd-d-flex',
+          'd-mb2',
+          'd-bar4',
+          'd-d-block',
+          'd-ta-left',
+          'd-fc-primary-inverted',
+          'dt-emoji-list-item',
+          'd-bgc-transparent',
+          'd-w100p',
```

Comment:
Move the utility classes to a standard css class using dialtone css variables instead. We avoid using utility classes anywhere that isn't the application level now.

Reply:
moved it.!